{"additions": 1, "auther_ref": "JPT7XX", "auther_sha": "7d9a2b58a43954dc4c955b92537cdd07209e5580", "author": "kodolova", "body": "JetPlusTrack correction practically does not work and return calojet energy only without adding ak4JetExtender to the python configurator.\n", "branch": "CMSSW_7_6_X", "changed_files": 1, "closed_at": "1441077695", "comments": 26, "commits": 4, "created_at": "1440661057", "deletions": 1, "labels": ["comparison-available", "fully-signed", "reconstruction-approved", "tests-approved"], "merge_commit_sha": "461927093ffa11a7a5b38739c03ee24a80179950", "merged_at": "1441077695", "merged_by": "cmsbuild", "milestone": "CMSSW_7_6_X", "number": 10977, "release-notes": [], "review_comments": 1, "state": "closed", "title": "Fix bug in JetPlusTrackCorrections_cff.py: add JetVertexExtender", "updated_at": "1441077695", "user": "kodolova"}